src/handlers/ps2census/MetagameEventEventHandler.ts

```diff
@@ -44,24 +43,21 @@ export default class MetagameEventEventHandler implements QueueMessageHandlerInt
         let instanceId = `${event.world}-${event.instanceId}`;
         let round = 0;
 
-        /* eslint-disable no-bitwise */
+        /* eslint-disable no-bitwise, @typescript-eslint/no-unsafe-assignment */
         if ((event.zone >> 16) !== 0 && (event.zone & 0xFFFF) === Zone.NEXUS) {
-            instanceId = `outfitwars-${event.world}-${Zone.NEXUS}-${(event.zone >> 16) & 0xFFFF}`;
-            await this.ps2AlertsApiClient.get(`${ps2AlertsApiEndpoints.outfitwarsRankings}?sortBy=round&order=desc&world=${event.world}`).then(
-                (value) => {
-                    if (!value.data) {
-                        throw new Error('No data received!');
-                    }
-
-                    const rankings: OutfitwarsRankingInterface[] = value.data as OutfitwarsRankingInterface[];
-                    round = rankings[0].round;
-                },
+            instanceId = `outfitwars-${event.world}-${Zone.NEXUS}-${event.instanceId}`;
+            const {
+                data: currentRound,
```

Comment:
This looks slightly strange, how does the data get assigned here?

Reply:
This is object destructuring: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

The api client returns a promise that has an object with a 'data' key which i destructure into the currentRound const